config.py

```diff
@@ -0,0 +1,7 @@
+# Not used
```

Comment:
Missed this in the earlier round -- I don't think this is going to work for the WSGI app, only modules under "kickoff" will be importable, can you move this to something inside of that directory?